DOC-7538: Mark channels as favorites page needs animated GIFs for mobile app workflows (#7546)
mattermost/docs: source/collaborate/favorite-channels.rst

.. tab:: Mobile

  1. Tap the channel you want to mark as a favorite.

  .. image:: ../images/mobile-select-a-channel.jpg
      :alt: Select a channel that you want to mark as favorite.
      :scale: 30

  2. Tap the |more-icon-vertical| icon located in the top right corner of the app.

  .. image:: ../images/mobile-select-more-options-for-a-channel.jpg
      :alt: Tap on More options to access available options for the channel.
      :scale: 30

  3. Tap **Favorite**.

  .. image:: ../images/mobile-select-view-info-for-a-channel.jpg
      :alt: Tap on Favorite to mark the channel as one of the favorites.
      :scale: 30

> .. note::

Reviewer: ```suggestion
.. tip::
```

>   You can also mark a channel as favorite in this way.

Reviewer: ```suggestion
  Alternatively, you can mark a favorite channel as follows:
```
